cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -98,6 +98,14 @@ def counts_to_results(
     return result
 
 
+class InvalidInputError(Exception):
```

Comment:
i think it's ok to just use the builtin `ValueError` for these - it's (i think) typically what's used for bad input values in python

Reply:
Yeah, was originally going to have a class that could be updated with various error codes to standardize error handling for the different methods so that it would be easier to maintain and update error messages later on. So a remnant of that.